12-offer-encoding.md

```diff
@@ -58,6 +58,19 @@ The merchant-pays-user flow (e.g. ATM or refund):
 3. The merchant confirms the *invoice_node_id* to ensure it's about to pay the correct
    person, and makes a payment to the invoice.
 
+The pay-mobile-user flow (e.g. paying a friend back to their mobile node):
+1. The mobile user supplies some always-online node with a static (i.e. 
+   `payment_hash`-less) invoice to return on its behalf. This always-online node may 
+   be the mobile user's channel counterparty, wallet vendor, or another node on the 
+   network that it has an out-of-band relationship with.
+2. The mobile user publishes an offer that contains blinded paths that terminate
+   at the always-online node.
+3. The payer sends an `invoice_request` to the always-online node, who replies
+   with the static invoice previously provided by the mobile user if the mobile user 
+   is offline. If they are online, the `invoice_request` is forwarded to the mobile 
+   user as usual.
```

Comment:
Why would you distinguish between the offline and online case - can't the offline flow be used always for simplicity?